File: Remove consecutive duplicate words/Remove consecutive duplicate words.js
function removeConsecutiveDuplicates(string) {
  const arr = string.split(" ");
  let result = [arr[0]]
  for(let i=1;i<arr.length;i++){
    if(arr[i]!=arr[i-1]){
      result.push(arr[i])
    }
  }
  return result.join(" ")
}

File: Remove consecutive duplicate words/Remove consecutive duplicate words.py
def remove_consecutive_duplicates(s : str) -> str:
    result = []
    arr = s.split(" ")
    result.append(arr[0])
    for i in range(1,len(arr)):
            if arr[i]!=arr[i-1]:
                result.append(arr[i])
    return " ".join(result)

